Assets/LeapMotion/Scripts/LeapImageRetriever.cs

```diff
@@ -89,6 +89,10 @@ public class LeapTextureData {
       TextureFormat format = getTextureFormat(image);
 
       if (_combinedTexture != null) {
+				if ((_combinedTexture.height == combinedHeight) &&
```

Comment:
You would need to check image format as well, since that can theoretically change even if the width/height remains the same.  However, this method should only be called when a reconstruct is needed, if it is being called every frame, that is a bug elsewhere and should be addressed elsewhere.  (I think it has been fixed, the stale flag wasn't being reset before).  

Since it is only going to be called infrequently (upon device connection / flip / swap) I would be hesitant to try adding complexity to this method to try to detect whether or not every property is the same, as it might cause difficult-to-track-down edge cases like this one.